nais/vars-p.yaml

```diff
@@ -10,6 +10,8 @@ abac_pdp_endpoint_url: https://wasapp.adeo.no/asm-pdp/authorize
 norg_url: https://app.adeo.no/norg2/ws/OrganisasjonEnhet/v2
 oppgave_url: https://oppgave.nais.adeo.no/api/v1
 isso_isalive_url: https://isso.adeo.no/isso/isAlive.jsp
+kafka_url: a01apvl00145.adeo.no:8443,a01apvl00146.adeo.no:8443,a01apvl00147.adeo.no:8443,a01apvl00149.adeo.no:8443
```

Comment:
@chriswk - kan du gjøre en QA av spesielt disse to verdiene? Vi hadde ikke lagt til noe for produksjon, og fant noe i et annet repo, men vet ikke om det er riktig.